app/src/main/java/in/testpress/testpress/authenticator/RegisterActivity.kt

```diff
@@ -0,0 +1,73 @@
+package `in`.testpress.testpress.authenticator
+
+import `in`.testpress.testpress.Injector
+import `in`.testpress.testpress.R
+import `in`.testpress.testpress.authenticator.LoginActivity.REQUEST_CODE_REGISTER_USER
+import `in`.testpress.testpress.core.TestpressService
+import `in`.testpress.testpress.enums.VerificationMethod
+import `in`.testpress.testpress.models.InstituteSettings
+import `in`.testpress.testpress.ui.fragments.AutoLoginFragment
+import `in`.testpress.testpress.ui.fragments.EmailVerificationFragment
+import `in`.testpress.testpress.ui.fragments.PhoneVerificationFragment
+import android.content.Intent
+import android.os.Bundle
+import androidx.annotation.VisibleForTesting
+import androidx.appcompat.app.AppCompatActivity
+import javax.inject.Inject
+
+open class RegisterActivity : AppCompatActivity() {
+
+    @Inject
+    lateinit var testPressService: TestpressService
+
+    private val instituteSettings = InstituteSettings.getInstance()
+
+    private val verificationMethod = instituteSettings.verificationType
```

Comment:
This is used in only one place so no need for instance variable, you can use it direclty

Reply:
You can put this in master